.github/workflows/storybook-release.yml

```diff
@@ -0,0 +1,34 @@
+name: storybook-release
+on: 
+  push:
+    branches: 
+      - main
+    tags:
+      - 'w-v*'
+    paths:
+      - 'chat-widget/**'
+
+permissions:
+  actions: write
+  contents: write
+  deployments: write
+  packages: write
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checking out for ${{ github.ref }}
+        uses: actions/checkout@v3
+      - name: Install Chat Widget packages
+        working-directory: chat-widget
+        run: yarn install
+      - name: Build Composite Storybook
+        working-directory: chat-widget
+        run: yarn build-composite-storybook
+      - name: Deploy Storybook
+        uses: JamesIves/github-pages-deploy-action@v4.3.3
+        with:
+          branch: gh-pages
+          folder: ./chat-widget/storybook-build
+          target-folder: omnichannel-chat-widget/storybook-build
```

Comment:
Would it be better if it was `omnichannel-chat-widget/docs` or something similar?

Reply:
How about stories or storybook?
Since this folder will contain the static build files for the storybook
We can maybe add docs later for adding some other documentations?